### DISCUSSION TITLE
Reframework with MHR (empress) crashes on start up

### DISCUSSION THREAD

**Discussion by AegisXI** [praydog/REFramework#642] [General Questions] [answered]
Hello, thank you for your work. I really love it and want to use. Could you help me solve this problem.

I run MHR with reframework but it always stop at Initialization and automatic shut down game.
I tried uninstall and rerun many times (both no mod and some fluffy 5000 mod) and still get this trouble. Windows defender, security, overlay program, etc... are all turn off.

I don't see "crash.dmp" file anywhere, so could give you only "log" file below.
[re2_framework_log.txt](https://github.com/praydog/REFramework/files/10675094/re2_framework_log.txt)

**Comment by praydog** [comment, marked as answer]
Pirated copies are not supported.
